automation/Commands/browser_commands.py

```diff
@@ -141,16 +141,13 @@ def extract_links(webdriver, browser_params, manager_params):
 
     sock.close()
 
-def browse_website(url, num_links, sleep, webdriver, proxy_queue,
+def browse_website(url, num_links, sleep, visit_id, webdriver, proxy_queue,
                    browser_params, manager_params, extension_socket):
     """
     calls get_website before visiting <num_links> present on the page
-    NOTE: top_url will NOT be properly labeled for requests to subpages
```

Comment:
This comment should not have been removed, as it's still true. The wording needs to be updated to show explain that the top_url in the site visits table won't match.